t/openapi-compatibility.t

```diff
@@ -0,0 +1,42 @@
+use Mojo::Base -strict;
+
+use File::Spec;
+use Mojo::Util;
+use Test::More tests => 3;
+
+ok(
+  module_calls_deprecated('JSON::Validator::OpenAPI'),
+  "JSON::Validator::OpenAPI->validate_request is deprecated"
+);
+
+ok(
+  !module_calls_deprecated('JSON::Validator::OpenAPI::Dancer2'),
+  "JSON::Validator::OpenAPI::Dancer2->validate_request isn't deprecated"
+);
+
+ok(
+  !module_calls_deprecated('JSON::Validator::OpenAPI::Mojolicious'),
+  "JSON::Validator::OpenAPI::Mojolicious->validate_request isn't deprecated"
+);
+
+sub module_calls_deprecated {
+  my $module = shift;
+
+  my $called = 0;
+  {
+    no warnings 'redefine';
+    *Mojo::Util::deprecated = sub { ++$called };
```

Comment:
`Mojo::Util` is a treasure of wonderful tools. After you suggest using functions from this package multiple times I've now read the whole POD to avoid getting into this again;-)